src/components/App/index.test.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import { render } from 'react-testing-library';
+import 'jest-styled-components';
```

Comment:
this should go in  `src/setupTests.js` as then we can avoid importing it in all our test files

Reply:
done